packages/react/src/components/InlineAlert/index.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import { Icon } from "..";
+import { INLINE_ALERT_TYPES, ICONS } from "../../constants/";
+
+const ALERT_ICONS = {
+  [INLINE_ALERT_TYPES.INFORMATION]: ICONS.IconInfo,
+  [INLINE_ALERT_TYPES.SUCCESS]: ICONS.IconCheck,
+  [INLINE_ALERT_TYPES.ERROR]: ICONS.IconStop,
+};
+
+const ALERT_TYPES = Object.values(INLINE_ALERT_TYPES);
+
+const InlineAlert = ({ title, children, type, onClose }) => (
+  <div
+    className={cx("InlineAlert", `InlineAlert--${type}`, {
+      "is-closeable": !!onClose,
+    })}
+  >
+    {onClose && (
+      <span className="InlineAlert-iconClose" onClick={onClose}>
+        <Icon icon={ICONS.IconClose} />
+      </span>
+    )}
+    <div className="InlineAlert-icon">
+      <Icon icon={ALERT_ICONS[type]} />
+    </div>
+    <div>
```

Comment:
Is this empty `<div />` needed?

Reply:
I'd say so. It's because of how I positioned the elements in containers.
Container with icon to the left, then **this** `<div>` with two values positioned vertically inside

Reply:
👍